src/types.ts

```diff
@@ -32,6 +35,7 @@ export type Arrow = {
   midPoints: Point[]
   toPoint: Point
   toMarker: Marker
+  color?: Color
```

Comment:
Why is color optional?

Reply:
if it's not there the arrow uses its `fromMarker`'s color. that way (after I fix the bug) when you change the color of the marker, the arrow updates as well. this happens in `src/components/ArrowLine.tsx:27`:
```ts
const color = useCssColor(arrow.color ?? arrow.fromMarker.color)
```